vendor/github.com/mindprince/gonvml/BUILD

```diff
@@ -4,6 +4,7 @@ go_library(
     name = "go_default_library",
     srcs = [
         "bindings.go",
+        "bindings_nocgo.go",
```

Comment:
hmm this seems wrong

Reply:
Why?

Reply:
because we do build kubelet with cgo enabled when building w/ bazel

Reply:
(see cgo=True below as well)

Reply:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/82342/pull-kubernetes-e2e-gce-device-plugin-gpu/1169320237401640960/ passed with a bazel build and GPU plugin tests so this must be working out fine ..., ignore me :upside_down_face:

Reply:
Yes, I build manually, with both CGO_ENABLED=0 and =1. Both flawless too :)

Reply:
yeah, it makes total sense with mak or `go build ...`, the bazel build just gets a bit weird (but hey, CI caching!) ...